src/TodoForm.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+
+class TodoForm extends React.Component {
+  state = {
+    currentValue: '',
+    currentChooseUserValue: '',
```

Comment:
```suggestion
    currentUser: null,
```
Я видел ты записываешь в это свойство число, почему тогда начальное значение - строка?